client/containers/Pub/PubSubmissionHeader/PubSubmissionHeader.stories.tsx

```diff
@@ -0,0 +1,72 @@
+import React from 'react';
+import { storiesOf } from '@storybook/react';
+
+import { SubmissionWorkflow } from 'types';
+
+import PubSubmissionHeader from './PubSubmissionHeader';
+
+const workflow: SubmissionWorkflow = {
+	id: '12089e31h3f23bf-f23f23f23f2-f23f32bu234v23-f2323f',
+	createdAt: '',
+	updatedAt: '',
+	enabled: true,
+	title: 'A Submission Work-work-work-work-work-work-flow',
+	collectionId: '',
+	targetEmailAddress: '',
+	emailText: {
+		type: 'doc',
+		attrs: {
+			meta: {},
+		},
+		content: [],
+	},
+	introText: {
+		type: 'doc',
+		attrs: {
+			meta: {},
+		},
+		content: [],
+	},
+	instructionsText: {
+		type: 'doc',
+		attrs: {
+			meta: {},
+		},
+		content: [
```

Comment:
This H1 content is actually the `submissionWorkflow.title` — it should be moved out of here and rendered using an `<h1>` in React in the `InstructionsTab`.